demo-ehr-app/src/App.css

```diff
@@ -0,0 +1,42 @@
+#root {
```

Comment:
Let's add apache licence for new source files

Reply:
Added. https://github.com/wso2/solutions-healthcare-demos/pull/2/commits/6e044a5fd78a9e997fccca0f49b0fe94a76abb9d